blueocean-core-js/src/js/i18n/i18n.js

```diff
@@ -0,0 +1,45 @@
+import i18n from 'i18next';
+import LngDetector from 'i18next-browser-languagedetector';
+import XHR from 'i18next-xhr-backend';
+
+/**
+ * Init language detector, we are going to use first queryString and then the navigator prefered language
+ */
+export const lngDetector = new LngDetector(null, {
+    // order and from where user language should be detected
+    order: ['querystring', 'navigator'],
+    // keys or params to lookup language from
+    lookupQuerystring: 'language',
+});
+/**
+ * configure the backend for our locale
+ */
+export const xhr = new XHR(null, {
+    loadPath: '/jenkins/i18n/resourceBundle?language={lng}&baseName={ns}',
```

Comment:
Does this path need to integrate with `UrlConfig` to get the proper "jenkins path"? Seems like this could break for deployments where standard "jenkins" context isn't used.